data/json/mapgen/deserter_city/deserter_city_office.json

```diff
@@ -0,0 +1,318 @@
+        "w.................#####/",
+        "#f................G...#/",
+        "#f................V.dhw/",
+        "w.................V^da#/",
+        "w.................#####/",
+        "#.h..h............G...#/",
+        "#^da^da...........V.dhw/",
+        "#######...........V^da#/",
+        "#T#T#T#...........#####/",
+        "#+#+#+#...........G...#/",
+        "#iiiii+...........V.dhw/",
+        "#CSCSC#...........V^da#/",
+        "#######...........VVVV#/",
+        "#CSCSC#...............#/",
+        "#iiiii+...............w/",
+        "#+#+#+#...............#/",
+        "#T#T#T#............<.Z#/",
+        "#######ww#####ww#######/",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_3fb",
+    "object": {
+      "fill_ter": "t_linoleum_gray",
+      "rows": [
+        "////////////////////////",
+        "########ww#####ww######/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#ZZZZZZZZZZZZZZZR.....#/",
+        "#wwwww#ZZZZZZZZZR.....#/",
+        "#^.h.^wZZZZZZZZZR.....#/",
+        "w..dd.wZZZZZZZZZR.....#/",
+        "#.....wZZZZZZZZZR.....w/",
+        "w.h.h.#RRRRRRRRRR.....#/",
+        "#.....V^.............h#/",
+        "w.....V..............n#/",
+        "#.....G..............hw/",
+        "w.....V...........##+##/",
+        "#fff..V^...hnh...^#>..#/",
+        "########ww#####ww######/",
+        "////////////////////////"
+      ],
+      "palettes": [ "deserter_city" ],
+      "monster": { ".": { "group": "GROUP_LAB_BASIC_SECURITY", "chance": 5 } }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "deserter_city_office_roofb",
+    "object": {
+      "fill_ter": "t_flat_roof",
+      "rows": [
+        "                        ",
+        "----------------------- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-............X........- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        "-.....................- ",
+        " -....................- ",
+        " -....................- ",
+        " -....................- ",
+        " -....................- ",
+        " -....................- ",
+        " -....................- ",
+        " -....&...............- ",
+        " -................AA..- ",
+        " -....................- ",
+        " ---------------------- ",
```

Comment:
This whole section is missing one column of roof tiles.
![kép](https://user-images.githubusercontent.com/44979050/236921235-bc120999-31e7-484e-9ce3-a3262203c727.png)
There are also no gutter drops for the rainwater to flow down.